### PR TITLE
DB seeds

### PR DESCRIPTION
**Pull request by takeller**
## What's this PR do? 
  - Adds seeds to the database
  - Command Line Script that adds an option on python3 run.py
     - run with --seed=True to wipe all campaign and request entries in the database and populate the database from seed file 
     - Executing python3 run.py without a flag defaults to not seeding

## Where should the reviewer start? 
  - src/seeds/seed.py

## How should this be manually tested? 
Fire up the server with python3 run.py --seed=True and visit localhost:3000/api/v1/campaigns

## Any background context? 
We may need to consider what we set the crypto address strings to in our seeds. Currently, they are random and don't point anywhere. 
The command line script uses the Click package. Confirm you have it installed with pipenv install Click

### CODE REVIEW

File: eth_crowdfund_be/eth_crowdfund_api/run.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 from src.app import create_app
 
 if __name__ == '__main__':
   env_name = os.getenv('FLASK_ENV')
   app = create_app(env_name)
   # run app
   app.run(port=3000)
+
+  with app.app_context():
+    add_seeds()
```
Comment (takeller) [inline eth_crowdfund_be/eth_crowdfund_api/run.py:12]: with app.app_context is needed to make the connection to psql

File: eth_crowdfund_be/eth_crowdfund_api/src/seeds/seed.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
+from ..models import db
+from ..models.Campaign import Campaign
+from ..models.Request import Request
+import click
+
+@click.command()
+@click.option('--seed', default=False, help='Seed database?')
```
Comment (takeller) [inline eth_crowdfund_be/eth_crowdfund_api/src/seeds/seed.py:7]: Allows us to seed DB by adding a flag to python3 run.py

File: eth_crowdfund_be/eth_crowdfund_api/src/seeds/seed.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        return 
+    
+    click.echo('Seeding...')
+
+#  Campaign Seeds
+    campaign1 = {
+        'name': 'Build a Well', 
+        'description': 'A small town in South Africa needs a well for clean water', 
+        # Uses lorem picsum for random pics
+        'image': 'https://picsum.photos/200/300',
```
Comment (takeller) [inline eth_crowdfund_be/eth_crowdfund_api/src/seeds/seed.py:19]: Used Lorem Picsum, but I can set to something more meaningful if needed.

File: eth_crowdfund_be/eth_crowdfund_api/src/seeds/seed.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    click.echo('Seeding...')
+
+#  Campaign Seeds
+    campaign1 = {
+        'name': 'Build a Well', 
+        'description': 'A small town in South Africa needs a well for clean water', 
+        # Uses lorem picsum for random pics
+        'image': 'https://picsum.photos/200/300', 
+        # Manager is a random string
+        'manager': 'X5JT498FJeklnsd8382hf',
```
Comment (takeller) [inline eth_crowdfund_be/eth_crowdfund_api/src/seeds/seed.py:21]: Currently random. We may need to update these to actually point somewhere

Comment (AntonioJacksonII) [reply eth_crowdfund_be/eth_crowdfund_api/src/seeds/seed.py:21]: I agree. I think we can have these point to an actual manager on the test network once we have those created.

**Review by AntonioJacksonII — APPROVED** [review]

**Review by perryr16 — APPROVED** [review]

**Pull request merged**